src/main/msp/msp.c

```diff
@@ -1116,6 +1117,17 @@ static bool mspProcessOutCommand(mspDescriptor_t srcDesc, int16_t cmdMSP, sbuf_t
             // config state flags - bits to indicate the state of the configuration, reboot required, etc.
             // other flags can be added as needed
             sbufWriteU8(dst, (getRebootRequired() << 0));
+
+            // Added in API version 1.46
+            // Write CPU temp
+            if (cmdMSP == MSP_STATUS_EX) {
+#ifdef USE_ADC_INTERNAL                
+                int16_t coretemp = getCoreTemperatureCelsius();
+#else
+                int16_t coretemp = 0;
+#endif                
+                sbufWriteU16(dst, coretemp);
+            }
```

Comment:
Why we have MSP_STATUS and MSP_STATUS_EX almost the same? What is the difference between both? I see the PID profile count vs gyro cycle time.  Something more? Why we don't send the coretemp for both?

Is not a problem of your code, only trying to understand.

Reply:
I can't say, believe its a matter of backward compability. But also wonder, if this are not an issue, since we have changed API to 1.46.
No problem from my point of view to send the coreTemp for both MSP_STATUS and MSP_STATUS_EX.

Reply:
I have changed, now no difference MSP_STATUS and MSP_STATUS_EX

Reply:
@haslinghuis hope you agree

Reply:
:thinking:  we should drop MSP_STATUS_EX and consolidate functionality using updated MSP_STATUS

@klutvott123 does this impact LUA? Thoughts?

MSP_STATUS_EX was introduced long time ago as MSP_STATUS is MultiWii compliant :confused: 

    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_32)) {
            MSP.send_message(MSPCodes.MSP_STATUS_EX, false, false);
        } else {
            MSP.send_message(MSPCodes.MSP_STATUS, false, false);
        }

Reply:
MultiWii Homepage look very obsolute ...